packages/components/src/styles/components/code-editor/index.scss

```
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// CODE-EDITOR
+//
+
+@import "theme";
```

```suggestion
@use "theme";
```
Small change to avoid the sass deprecation warnings, and align with other scss files.